lib/config.ts

```diff
@@ -242,8 +242,10 @@ export function get(name: string): any
     };
 
     // Restify bodyParser plugin options
-    otherConf['bodyParserOptions']
-        = { maxBodySize: convictConf.get('maxBodySize') };
+    otherConf['bodyParserOptions'] = {
+        maxBodySize: convictConf.get('maxBodySize'),
+        mapParams: false
```

Comment:
`mapParams` tells restify whether to dump the parse results into the `params` of the request in addition to `body` or `query` (or elsewhere, but right now we're only using those two). Setting it to false means we have to look specifically in the right place, rather than just checking `params` for everything, but also minimizes collisions.